test/unit/recipe/emailpassword/emailPassword.test.tsx

```diff
@@ -626,4 +626,18 @@ describe("EmailPassword", function () {
             }
         }
     });
+
```

Comment:
Modfy changelog and version as well - since this is a non breaking change, it can be 0.35.8 (there is already 0.35.7 that is in the process of being released)

Reply:
done